Make sure .unity3d has its damn .dat lines (#74)
unturned-info/unturned-3-knowledgebase: docs/Modding/AssetBundles.md

3. Navigate to the Bundles/Sources directory.
4. Import the Project.unitypackage.

## Individual Asset Bundles (*.unity3d)

> Most official files have transitioned to the master bundle system, but some uses still exist like the per-map road textures and testing individual assets.

Reviewer: ```suggestion
Most official files have transitioned to the master bundle system, but some uses still exist, such as the per-map road textures and testing individual assets.
```

### Required \*.dat File Lines

> __Asset\_Bundle\_Version__: `2, 3, 4`. Used to determine shader consolidation. Version 2 is Unity 2017 LTS. Version 3 is Unity 2018 and 2019 LTS. Version 4 is Unity 2020 LTS.

Reviewer: ```suggestion
__Asset\_Bundle\_Version__: `2, 3, 4`. Used to determine shader consolidation. Version 2 is Unity 2017 LTS, version 3 is Unity 2018 and 2019 LTS, and version 4 is Unity 2020 LTS.
```

!!! warning
    The game no longer loads .content files. MasterBundles now support all the features of a .content file, and the .content file tooling has been removed from Project.unitypackage

> This format was originally used by devkit landscapes, material palettes and radio songs, but has now been replaced by masterbundles.

Reviewer: ```suggestion
This format was originally used for devkit landscapes, material palettes, and radio songs, but it has now been replaced by masterbundles.```

!!! warning
    The game no longer loads .content files. MasterBundles now support all the features of a .content file, and the .content file tooling has been removed from Project.unitypackage

> This format was originally used for devkit landscapes, material palettes, and radio songs, but it has now been replaced by masterbundles.```

Reviewer: Is the code block formatting intended to be there?

Author: I didn't put that there.
